apps/api_web/config/config.exs

```diff
@@ -7,13 +7,18 @@ import Config
 
 # Configures the endpoint
 config :api_web, ApiWeb.Endpoint,
+  adapter: Bandit.PhoenixAdapter,
   url: [host: "localhost"],
   root: Path.dirname(__DIR__),
   secret_key_base: "v1EHfW07QPr8ai7bi0hooadtBorROPNjhSWx7CGv7AiCOhEyGoeT1jagMTNCE3PU",
   render_errors: [accepts: ~w(json html)],
   http: [
-    compress: true,
-    protocol_options: [idle_timeout: 86_400_000, request_timeout: 120_000]
+    http_1_options: [
```

Comment:
Was this intentionally only set for `http_1_options`? (https://hexdocs.pm/bandit/Bandit.html#t:http_2_options/0)

Reply:
We don't receiving incoming HTTP2 connections from the ALB, so it wouldn't be used.